### PR TITLE
Svi update fdo_sys

### PR DESCRIPTION
**Pull request by rftemple**
Added Fdo_sys module support to client and server

### CODE REVIEW

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java [hunk 1]
```diff
@@ -1,50 +1,112 @@
...
 import org.fidoalliance.fdo.protocol.message.ServiceInfoModuleState;
+import org.fidoalliance.fdo.protocol.message.ServiceInfoQueue;
+import org.fidoalliance.fdo.protocol.message.To2AddressEntries;
+import org.fidoalliance.fdo.protocol.serviceinfo.DevMod;
 import org.fidoalliance.fdo.protocol.serviceinfo.FidoAlliance;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
 
-public class ConformanceDeviceModule implements ServiceInfoModule {
+public class ConformanceOwnerModule implements ServiceInfoModule {
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java:25]: class name isn't matching the file name. (ConformanceDeviceModule.java)

**Review by DukeDavis12 — APPROVED** [review]

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java [hunk 2]
```diff
@@ -55,8 +60,11 @@ public void receive(ServiceInfoModuleState state, ServiceInfoKeyValuePair kvPair
    
           }
         }
+<<<<<<< HEAD
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java:63]: Please remove conflict tags ` <<<<<<< HEAD`

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java [hunk 3]
```diff
@@ -1,50 +1,120 @@
...
+import org.hibernate.Transaction;
 
-public class ConformanceDeviceModule implements ServiceInfoModule {
+public class ConformanceOwnerModule implements ServiceInfoModule {
 
+<<<<<<< HEAD
+=======
   private final LoggerService logger = new LoggerService(DeviceApp.class);;
+
+>>>>>>> added fdo_sys modlue
```
Comment (johneasterday) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java:31]: some unfinished merge conflict resolution

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java [hunk 4]
```diff
@@ -1,50 +1,120 @@
...
+   
+          }
         }
+<<<<<<< HEAD
+      }
+      break;
+=======
+>>>>>>> added fdo_sys modlue
+      default:
+        break;
```
Comment (johneasterday) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java:69]: another merge conflict

**Review by DukeDavis12 — APPROVED** [review]

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java (hunk 4 above)
Comment (rftemple) [reply component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java:43]: this was fix in commit a0cbcb9f3a1801f6e4631c23d5803face1f4e9ce

**Comment by rftemple** [comment]
merging poll request

**Pull request merged**